src/api/app/models/project.rb

```diff
@@ -197,6 +197,39 @@ def remote_image_template_from_xml(remote_project, image_template_project)
       project
     end
 
+    def package_templates
+      Project.with_package_templates + remote_package_templates
+    end
+
+    def remote_package_templates
+      result = []
+      Project.remote.each do |project|
```

Comment:
We're inside `Project` right? You don't need the class name...

Reply:
Removed